### PR TITLE
Custom loss addition in create and compile 

### PR DESCRIPTION
**Pull request by 123Soham-bhatia**
I have created custom loss edition in create and compile model function in the utils.py file as well as updated the contributing.md file for this also.

### CODE REVIEW

File: CONTRIBUTING.md
Comment (JoaquinAmatRodrigo) [inline CONTRIBUTING.md:1]: CONTRIBUTING.md should not be modified

File: skforecast/deep_learning/utils.py [hunk 1]
```diff
@@ -213,22 +241,103 @@ def create_and_compile_model(
...
+            warnings.warn("`loss` passed in `compile_kwargs`. Using the loss parameter instead.")
 
         model.compile(optimizer=optimizer, loss=loss, **compile_kwargs)
 
-    return model
\ No newline at end of file
+    return model
+
+
+# Unit tests
```
Comment (JoaquinAmatRodrigo) [inline skforecast/deep_learning/utils.py]: Unit tests should be located in the tests folder of each module
